docs/results.md

```diff
@@ -332,8 +332,8 @@ Gitlab SAST reports are sorted by severity (from high to info), following [Gitla
 	"vulnerabilities": [
 		{
 			"id": "32e763ac363dfee1ea972d951fb3de00f5f7a8d3f9f57b93e55e2d51957794a6",
-			"category": "Insecure Configurations",
```

Comment:
It's a shame to lose this information, I'd suggest either appending it to either the name or message.

Reply:
`Category` is really an enum type? [Looking at the schema](https://gitlab.com/gitlab-org/security-products/security-report-schemas/-/raw/v13.1.0/dist/sast-report-format.json), category is defined as follow:
```json
"category": {
  "type": "string",
  "minLength": 1,
  "description": "Describes where this vulnerability belongs (for example, SAST, Dependency Scanning, and so on)."
}
```

Also, am I using the right scheme?

Reply:
@felipe-avelar I believe it should be an enum and this is a bug, which is why I opened https://gitlab.com/gitlab-org/gitlab/-/issues/332054 to fix this and notify the team. It has been overlooked previously, but this essentially drives our groupings within the UI:

![Screen Shot 2021-05-27 at 9 38 04 AM](https://user-images.githubusercontent.com/1449707/119864258-4fb98c80-becf-11eb-81ad-cd0c65bf59d9.png)

We talk more about the field [on our integration page](https://docs.gitlab.com/ee/development/integrations/secure.html#category)

> Also, am I using the right schema?

You are! This is just a bug that I noticed on glancing over the diff, so it's on our end.

We should be able to better support different categorizations in the future beyond report types and are currently exploring doing so via CWE mapping instead: https://gitlab.com/gitlab-org/gitlab/-/issues/300978, but for now the grouping relies on the analysis type to differentiate between SAST, Dependency Scanning, DAST, Fuzzing, etc.

Reply:
@theoretick Thanks for your response. ❤️ 
we will think where we should put category information.

Also, thanks for your contribution 😄

Reply:
Happy to help! This is actually my last day before taking a code-free vacation so I won't be around until around June 7th.

I'll poke my colleague to keep an eye on this one however if any follow-ups are needed after EOD